checkstyle/import-control-coordinator-common.xml

```diff
@@ -40,6 +40,9 @@
 
     <subpackage name="coordinator">
         <subpackage name="common">
+
+            <allow pkg="org.apache.kafka.common" />
```

Comment:
Line number 34 - 39 above defines common classes. Which common package do we need explitily, can we please include that above.